dapp/src/routes/DnsVerification/DnsVerification.svelte

```diff
@@ -0,0 +1,207 @@
+    >
+      <div
+        class="mb-4 text-2xl text-left font-bold body flex flex-row items-center"
+      >
+        <div class="mr-3">DNS Verification</div>
+      </div>
+      <div class="body">
+        This process is used to link your web domain to your Tezos account by
+        entering your domain, signing a message using your private key, entering
+        the information into the TXT, and finally retrieving that data to
+        verify.
+      </div>
+    </div>
+
+    <VerificationStep
+      step={1}
+      bind:currentStep
+      title="Enter Domain"
+      description="Enter the domain you want want to verify ownership for."
+    >
+      <div class="flex w-full mt-8">
+        <Input
+          placeholder="Enter domain"
+          class="mr-8"
+          bind:value={domainUrl}
+          disabled={currentStep !== 1}
+          name="enter-domain"
+        />
+        {#if currentStep === 1}
+          <PrimaryButton
+            text="Submit"
+            onClick={async () => {
+              next(async () => {
+                try {
+                  dnsClaim = getPreparedUnsignedDnsMessage(
+                    domainUrl,
+                    $userData
+                  );
+                } catch (err) {
+                  alert.set({
+                    variant: 'error',
+                    message: `Failed to create DNS claim: ${
+                      err?.message || JSON.stringify(err)
+                    }`,
+                  });
+                }
+              });
+            }}
+            class="ml-4 lg:ml-0"
+            small
+          />
+        {/if}
+      </div>
+    </VerificationStep>
+
+    <VerificationStep
+      step={2}
+      bind:currentStep
+      title="Sign Prompt"
+      description="Sign the message presented to you containing your domain."
+    >
+      {#if currentStep >= 2}
+        <div class="flex items-center w-full py-2 mt-8">
+          <textarea
+            class="overflow-x-auto rounded-lg bg-gray-100 body p-2 mr-4 w-full resize-none"
+            bind:value={dnsClaim}
+            readonly
+            disabled
+          />
+          <CopyButton text={dnsClaim} />
+        </div>
+      {/if}
+      {#if currentStep === 2}
+        <PrimaryButton
+          text="Sign Message"
+          class="mt-8 lg:w-48"
+          onClick={async () => {
+            next(async () => {
+              dnsMessage = await getDnsFullSignedClaim(
+                $userData,
+                domainUrl,
+                $wallet
+              );
+            });
+          }}
+          disabled={lock}
+        />
+      {/if}
+    </VerificationStep>
+
+    <VerificationStep step={3} bind:currentStep title="Add Record to Your DNS">
+      <div class="body">
+        Add this signed message to your DNS profile as a TXT entry. The DNS
+        provider might take a few minutes to update.
```

Comment:
```suggestion
    <VerificationStep step={3} bind:currentStep title="Upload TXT Record">
      <div class="body">
        Create a new TXT record for `@` and put the following text as the value.
        Keep in mind that DNS propagation can take a few minutes.
```

Or something along these lines. Have a look at the documentation of other services like ProtonMail or Keybase which also make use of TXT records. Make sure you use the right terminology, e.g. "DNS profile" doesn't really make sense.

And the notice about DNS propagation, I'd say it could be removed and only displayed if the verification fails (because no record was found).